.github/workflows/packages.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
     - name: Install dependencies
-      run: sudo apt-get update; sudo apt-get -y install build-essential git ncurses-dev 
+      run: sudo apt-get update; sudo apt-get -y install build-essential git ncurses-dev python2
```

Comment:
This was only required for Turris OS 5.x, well, it was requirement by OpenWrt 19.07, which was using Turris OS 5.x, but for some time, you are using already Turris OS 6.x (https://github.com/CESNET/Nemea-OpenWRT/commit/22b8a455a31825c508362c92562c7ed251de6bd3), then this could be dropped. 

Reference:
https://github.com/openwrt/openwrt/commit/19938c8de7a062626796f53a2805608c0dd4edbd

Reply:
https://github.com/CESNET/Nemea-OpenWRT/pull/40